specification/testbase/resource-manager/Microsoft.TestBase/preview/2023-11-01-preview/actionrequest.json

```diff
@@ -0,0 +1,375 @@
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.TestBase/testBaseAccounts/{testBaseAccountName}/actionRequests/{actionRequestName}": {
+      "get": {
+        "tags": [
+          "Test Base Action Requests"
+        ],
+        "operationId": "ActionRequests_Get",
+        "description": "Get the action request under the specified test base account.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./testbase.json#/parameters/TestBaseAccountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ActionRequestName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Succeeded to get the action request under the specified test base account.",
+            "schema": {
+              "$ref": "#/definitions/ActionRequest"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ActionRequestGet": {
+            "$ref": "./examples/ActionRequestGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Test Base Action Requests"
+        ],
+        "operationId": "ActionRequests_Put",
+        "description": "Create (submit) an action request.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./testbase.json#/parameters/TestBaseAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ActionRequestName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "actionRequest",
+            "schema": {
+              "$ref": "#/definitions/ActionRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Succeeded to create (submit) an action request.",
```

Comment:
>create [](http://example.com/codeflow?start=41&length=6)

You'd want to fix this description - 200 should imply updating, not creating

Reply:
fixed